test/e2e/storage/volume_metrics.go

```diff
@@ -62,6 +66,13 @@ var _ = SIGDescribe("[Serial] Volume metrics", func() {
 		var err error
 
 		controllerMetrics, err := metricsGrabber.GrabFromControllerManager()
+
+		if err != nil {
+			if strings.Contains(err.Error(), "Kubelet is not registered") {
```

Comment:
Inspecting error message is really fickle. Can we feature flag this test instead?

Reply:
ack, I don't like it either. pushing a fix.

Reply:
fixed PTAL